modules/sievefilters/modules.php

```diff
@@ -34,7 +34,7 @@ public function process() {
             $this->out('conditions', json_encode(base64_decode($base64_obj)));
             $base64_obj = str_replace("# ", "", preg_split('#\r?\n#', $script, 0)[2]);
             $this->out('actions', json_encode(base64_decode($base64_obj)));
-            if (mb_strstr($script, 'allof')) {
+            if (strstr($script, 'allof')) {
```

Comment:
Is this intentional?

I am wondering because of https://github.com/cypht-org/cypht/issues/1224

Reply:
We should keep multibyte functions in 99% of the places. There was only one exception when the size of a string was specified in bytes in the IMAP protocol (literals).